bin/mongodb/update-medley-tour.js

```diff
@@ -4,11 +4,15 @@ db.tournament2
   .forEach(t => {
     print('updating medley tournament ' + t._id + '; ' + t.name);
     var intervals = Math.ceil(t.minutes / t.mMinutes);
+    var remainder = intervals * t.mMinutes - t.minutes;
+    var speeds = Array.from({ length: intervals }, (_, i) =>
+      i !== intervals - 1 ? NumberInt(t.mMinutes) : NumberInt(t.mMinutes - remainder)
```

Comment:
Think this needs to be seconds not minutes?

Reply:
yes this should be in seconds! *60